geo-traits/src/line_string.rs

```diff
@@ -0,0 +1,54 @@
+use geo_types::{Coord, CoordNum, LineString};
+
+use super::point::PointTrait;
+use std::iter::Cloned;
+use std::slice::Iter;
+
+pub trait LineStringTrait<'a>: Send + Sync {
+    type ItemType: 'a + PointTrait;
+    type Iter: Iterator<Item = Self::ItemType>;
+
+    /// An iterator over the points in this LineString
+    fn points(&'a self) -> Self::Iter;
```

Comment:
Thoughts on calling this (and the other methods) `coords`? Since, at least in the `geo` world, a `LineString` is made up of "coordinates" and not "points".

Reply:
Makes sense. I originally didn't have a `CoordTrait` and so had these named `point*` and having the item be an instance of `PointTrait` instead of `CoordTrait`